posts/2021-07-22-cloud-hosted-guides-post.adoc

```diff
@@ -65,23 +59,26 @@ image::/img/blog/OL_Cloud-hosted-3.png[Image 3,width=70%,align="center"]
 
 Once you are logged in you will then be directed to the landing page for the guide you wish to learn. Simply click on the “Access cloud-hosted guide” button and after a few minutes your environment should be set-up and ready to use.
 
-== The Basics of the Environment
+== What is the Skills Network and why are we using it with our guides?
```

Comment:
We don't need to mention Skills Network.

Reply:
Done